targets/TARGET_NORDIC/TARGET_NRF5/TARGET_MCU_NRF52840/TARGET_NRF52840_DK/PinNames.h

```diff
@@ -1,28 +1,28 @@
-/* 
```

Comment:
whitespaces create lot of noise, should be in separate commit, or even own pull request